.github/workflows/coverage.yml

```
@@ -52,21 +52,21 @@ jobs:
           - 1025:1025
     steps:
     - name: Checkout
-      uses: actions/checkout@v4
+      uses: actions/checkout@34e114876b0b11c390a56381ad16ebd13914f8d5 # v4
 
     - name: Get Composer Cache Directory
       id: composer-cache
       run: |
```

actionlint (1.7.11)

[error] workflow command "set-output" was deprecated. use `echo "{name}={value}" >> $GITHUB_OUTPUT` instead: https://docs.github.com/en/actions/using-workflows/workflow-commands-for-github-actions

(deprecated-commands)